VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -506,16 +506,27 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
     }
 
     getPanelRowTooltipText(participant: PanelModel) {
+        let toolTipText: string;
+
+        if (!participant.isDisconnected() && !participant.isInHearing()) {
+            toolTipText = participant.displayName + ': ' + this.getTranslatedText('not-joined') + this.getAdditionalText(participant);
+        }
         if (participant.isAvailable()) {
-            return participant.displayName + ': ' + this.getTranslatedText('joining') + this.getAdditionalText(participant);
+            toolTipText = participant.displayName + ': ' + this.getTranslatedText('joining') + this.getAdditionalText(participant);
         }
-        if (!participant.isDisconnected() && !participant.isInHearing()) {
-            return participant.displayName + ': ' + this.getTranslatedText('not-joined') + this.getAdditionalText(participant);
+        if (participant.isWitness && participant.isAvailable() && !participant.isInHearing()) {
+            toolTipText =
+                participant.displayName + ': ' + this.getTranslatedText('participant-available') + this.getAdditionalText(participant);
         }
         if (participant.isDisconnected()) {
-            return participant.displayName + ': ' + this.getTranslatedText('disconnected') + this.getAdditionalText(participant);
+            toolTipText = participant.displayName + ': ' + this.getTranslatedText('disconnected') + this.getAdditionalText(participant);
         }
-        return participant.displayName + this.getAdditionalText(participant);
+
+        if (!toolTipText) {
+            toolTipText = participant.displayName + this.getAdditionalText(participant);
```

Comment:
we can use this to initialize the toolTipText property, removes the need for this if block

Reply:
that's done now